extension/load-browser-state.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+
+async function populateManifestViews(manifest, dataLoader) {
+
+  let views = {}
+  for (let k of ['Answer', 'WebPage', 'Question', 'VideoObject', 'Product']) {
+    klass = manifest.findSchemaByName(k).entityClass();
+    view = manifest.newView(klass.type.viewOf(), k+'View');
+
+    views[k] = view;
+  }
+
+  let entities = await dataLoader();
+
+  dumpEntities(views, entities);
+}
+
+function dumpEntities(views, entityData) {
+  
+  for (let ei of entityData) {
```

Comment:
I'm not sure what `ei` is, try to avoid acronyms in variable names.

Reply:
Thanks, good reminder (it was meaningful in a previous codebase but not here).

I ended up removing this code entirely.